features/roomlist/impl/src/main/kotlin/io/element/android/features/roomlist/impl/RoomListPresenter.kt

```diff
@@ -116,10 +119,12 @@ class RoomListPresenter @Inject constructor(
                     }
                     displaySearchResults = !displaySearchResults
                 }
-                is RoomListEvents.ShowContextMenu -> {
+                is RoomListEvents.ShowContextMenu -> coroutineScope.launch {
+                    val isDm = client.getRoom(event.roomListRoomSummary.roomId).use { it?.isDm.orFalse() }
```

Comment:
Do you think it would be useful to add a field `isDm` to  `RoomListRoomSummary` instead?

Reply:
Yes it'd be a good idea to expose it in `RoomSummary` first, as we create this with a `RoomInfo` we have already everything we need for this.

Reply:
I didn't realise we were building it there, I'll add it there then. I wonder if exposing `isDm` and `isOneToOne` separately actually makes sense. Do we really want to handle something differently if the room is direct but has 3-4 people in it? AFAICT we only distinguish between rooms and DMs.